Validation/EcalDigis/src/EcalSelectiveReadoutValidation.cc

```diff
@@ -1378,7 +1378,13 @@ EcalSelectiveReadoutValidation::analyzeTP(edm::Event const & event,
     double etSum = ttEtSums[iEta0][iPhi0];
 
     int iE = meTp_->getTProfile()->FindFixBin(tpEt);
-    ++tpEtCount[iE];
+    if ((iE >= 0) && (iE < 100)) {
+      ++tpEtCount[iE];
+    } else {
+      // FindFixBin might return an overflow bin (outside tpEtCount range).
+      // To prevent a memory overflow / segfault, these values are ignored.
+      std::cout << "EcalSelectiveReadoutValidation: Invalid iE value: " << iE << std::endl;
```

Comment:
I thought, we shouldn't use `std::cout`. Shouldn't this be converted to message logger?

I see this is quite popular issue:
```
Begin processing the 88th record. Run 1, Event 7388, LumiSection 74 at 22-Jun-2015 15:37:19.177 CEST
EcalSelectiveReadoutValidation: Invalid iE value: 101
EcalSelectiveReadoutValidation: Invalid iE value: 101
EcalSelectiveReadoutValidation: Invalid iE value: 101
EcalSelectiveReadoutValidation: Invalid iE value: 101
```

@dmitrijus @deguio @Dr15Jones